rosidl_typesupport_c/include/rosidl_typesupport_c/type_support_map.h

```diff
@@ -22,15 +22,26 @@ extern "C"
 {
 #endif
 
+/// Standard structure for encapsulating information about a rosidl interface type's support.
 typedef struct type_support_map_t
 {
   // TODO(dirk-thomas) const should not be defined for the fields
   // but should be set for the struct when it is being used
   // same for rosidl_message_type_support_t et al
+
+  /// Number of type_supports in the data array.
   const size_t size;
+
+  /// The ROS 2 package this is generated from.
   const char * package_name;
+
+  /// Array of identifiers for the type_supports.
   const char * const * typesupport_identifier;
+
+  /// Names of the symbols associated with each type support.
   const char * const * symbol_name;
+
+  /// Type erased data pointer.
```

Comment:
The documentation should describe what data is behind these pointers.

Reply:
Fixed here and in _cpp.

Reply:
The doc should mentioned what kind of pointer is being stored. I think it is a `rcpputils::SharedLibrary` once a library is loaded.